lnd-rpc/src/main/scala/org/bitcoins/lnd/rpc/LndRpcClient.scala

```diff
@@ -131,8 +131,12 @@ class LndRpcClient(val instance: LndInstance, binaryOpt: Option[File] = None)(
       case None         => None
     }
 
+    val host =
+      if (instance.rpcUri.getHost == "0.0.0.0") "localhost"
```

Comment:
So it will never be able to bind to `0.0.0.0`? 

IIUC this correctly it will substitute `0.0.0.0` -> `localhost`?

Reply:
This is if it is bound to 0.0.0.0, we don't attempt to make requests to it, we instead make it to local host